pkg/config/analysis/incluster/controller.go

```diff
@@ -63,7 +63,7 @@ func NewController(stop <-chan struct{}, rwConfigStore model.ConfigStoreControll
 			Remove(collections.K8SApiextensionsK8SIoV1Customresourcedefinitions)
 	}
 
-	store, err := crdclient.NewForSchemas(kubeClient, "default",
```

Comment:
this hard-coded revision definitely looks like upstream is affected, too. except if they only ever want this to work on the default revision?